### PR TITLE
Added the performance entry buffer

### PR DESCRIPTION
**Pull request by plehegar**
Initial proposal to deal with getEntries* and clear*

### CODE REVIEW

**Comment by plehegar** [comment]
See also https://github.com/w3c/resource-timing/pull/47

**Comment by plehegar** [comment]
See also https://github.com/w3c/user-timing/pull/13

File: index.html [hunk 1]
```diff
@@ -324,7 +329,7 @@
         the _value_ of that [dictionary member].
         </li>
         <li>For each <a>PerformanceEntry</a> object
-        (<dfn>entryObject</dfn>) associated with the [context object], in
+        (<dfn>entryObject</dfn>) in the <a>performance entry buffer</a>, in
```
Comment (yoavweiss) [inline index.html]: Where are we adding entries to the performance entry buffer?

Comment (yoavweiss) [inline index.html]: found it. nm

**Comment by toddreifsteck** [comment]
LGTM. Merging

**Pull request merged**